app/src/main/java/com/sunasterisk/youtubebackground/utils/VideoUtils.kt

```diff
@@ -0,0 +1,22 @@
+package com.sunasterisk.youtubebackground.utils
+
+object VideoUtils {
+    fun formatLikeCount(likeCount: Long?): String{
+        if (likeCount == null) {
+            return "00000"
+        }
```

Comment:
Tương đương với dòng này nhé 
```suggestion
        linkCount ?: return "00000"
```